### ISSUE TITLE
Convo.say not working with Event API

### ISSUE THREAD

**Issue by airlikidh2** [howdyai/botkit#737]
Slack related:
So I spent 4 hours figuring why my convo.say was not triggered while I was trying to transition from RTM to Events API.

If using the Event API only, you need to call:
`controller.startTicking();`

Could this be added in the doc?
https://github.com/howdyai/botkit/blob/master/readme-slack.md#events-api

**Comment by peterswimm** [comment]
@airlikidh2 care to submit the PR?

**Comment by Bunk** [comment]
Thanks for this, I was experiencing the exact same issue.

Went ahead and submitted a documentation PR (#802) for you.

**Comment by lpedromachado** [comment]
I spent several hours too trying to make `convo.say()` work. It was only after I wrote an issue and I was typing its title that I saw this one. Now it works.

I see nowhere in the documentation either of `bot.startConversation()` or `convo.say()` an instruction to use `controller.startTicking()` first. Even the section about `startTicking()` doesn't say where to put it. I think the docs should be clearer about this.
